internal/store/fleet.go

```diff
@@ -124,6 +126,54 @@ func fleetSelectStr(withDeviceCount bool) string {
 		"*")
 }
 
+// ListRolloutDeviceSelection attempts to get all relevant fleets for rollout device selection.
+// A relevant fleet contains at least 1 device that at least one of the conditions below is true:
+// - marked as selected for rollout
+// - the template version of the fleet is not the same the template version in the annotation 'device-controller/renderedTemplateVersion'
+// - the field 'status.config.renderedVersion' is not the same as the annotation 'device-controller/renderedVersion'
+func (s *FleetStore) ListRolloutDeviceSelection(ctx context.Context, orgId uuid.UUID) (*api.FleetList, error) {
+	var fleets model.FleetList
+	err := s.db.Raw(fmt.Sprintf(`select * from (select *, annotations ->> '%s' as tv from fleets) as main_query
```

Comment:
I know you have already considered this but I am just passing along the article from go team.

https://go.dev/doc/database/sql-injection

Reply:
Just note that all parameters passed to the `Sprintf` function are constants.

Reply:
thanks